src/Samples/Sarif.Sdk.Sample/packages.config

```diff
@@ -2,7 +2,9 @@
 <packages>
   <package id="CommandLineParser" version="2.2.1" targetFramework="net461" />
   <package id="Newtonsoft.Json" version="11.0.2" targetFramework="net461" />
```

Comment:
>Newtonsoft.Json" version="11.0.2"  [](start = 15, length = 34)

Larry please move this down to 10.0.3 to match the SDK.

Reply:
@EasyRhin0  Thanks, fixed.